src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -616,6 +616,8 @@ function root_water_flux_per_ground_area!(
     ψ_soil::FT = s.ψ(t)
     fa .= FT(0.0)
     @inbounds for i in 1:n_root_layers
+        local_area_index = getproperty(area_index, model.compartment_labels[1])
```

Comment:
@kmdeck can you please comment on a good variable name for this?

(This occurs in a couple of other places)

Reply:
ah. this is the area index of the first compartment (above ground), which can be stem (e.g. trees) or leaf (e.g. grass). it can be outside of the loop, I guess. 

maybe...`above_ground_area_index`? im a little stuck. im fine with a wordy name if it's clear

Reply:
I went with `above_ground_area_index` :)